manifests/microshift/model-server-deployment.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kepler-model-server-cfm
+  namespace: kepler
+data:
+  MODEL_PATH: models
+  PROM_SERVER: 'http://kepler-exporter.kepler.svc.cluster.local:9102'
+  PROM_QUERY_INTERVAL: '20'
+  PROM_QUERY_STEP: '3'
+  PROM_HEADERS: ''
+  PROM_SSL_DISABLE: 'true'
+  INITIAL_MODELS_LOC: https://raw.githubusercontent.com/sustainable-computing-io/kepler-model-server/main/tests/test_models
+  INITIAL_MODEL_NAMES: |
+    AbsComponentModelWeight=KerasCompWeightFullPipeline
+    AbsComponentPower=KerasCompFullPipeline
+    DynComponentPower=ScikitMixed
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kepler-model-server
+  namespace: kepler
+  labels:
+    app.kubernetes.io/component: model-server
+    app.kubernetes.io/name: kepler-model-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: model-server
+      app.kubernetes.io/name: kepler-model-server
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: model-server
+        app.kubernetes.io/name: kepler-model-server
+    spec:
+      volumes:
+      - name: cfm
+        configMap:
+          name: kepler-model-server-cfm
+      - emptyDir: {}
+        name: models
+      securityContext: {}
+      serviceAccount: kepler-sa
+      serviceAccountName: kepler-sa
+      containers:
+      - name: server-api
+        image: quay.io/sustainable_computing_io/kepler_model_server:latest
+        imagePullPolicy: Always
+        resources:
+          requests:
+            cpu: 100m
+            memory: 400Mi
+        securityContext:
+          privileged: true
+        ports:
+        - containerPort: 8100
+        volumeMounts:
+          - name: cfm
+            mountPath: /etc/config
+            readOnly: true
+          - mountPath: /models
+            name: models
+        command: ["python3.8",  "model_server.py"]
```

Comment:
@rootfs FYI. I noticed we are using `python script_name.py` as commands in the pods definitions. This will buffer output by default and the stdout logs will not be displayed as they happen. To achieve unbuffered stdout on a container executing python directly as command we should be using the `-u` option like `python -u script_name.py` for unbuffered stdout.

Reply:
Thanks for the note, cc @sunya-ch @KaiyiLiu1234 on model server side.

Reply:
@williamcaban with recent updated the model-server deployment is no longer necessary - also, the MicroShift deployment no longer requires special consideration, so no longer including separate MicroShift manifests.